Gemfile

```diff
@@ -58,3 +58,4 @@ gem 'net-pop'
 gem 'net-smtp'
 
 gem 'carrierwave'
+gem 'i18n_generators'
```

Comment:
公式README読んでみましたか? はじめて使うgemは目を通すように心がけましょう。
https://github.com/amatsuda/i18n_generators#installation に

> Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.

は対応したいです。

Reply:
最後まで目を通していませんでした…！
今後気をつけます🙇‍♂️

Reply:
読んでいたのならこういった漏れは仕方ないので大丈夫です :+1:  全く読んでないとしたら駄目かなと思いますが。今後も引き続き目を通すようにしましょうー。